spec/rubocop/cop/performance/redundant_equality_comparison_block_spec.rb

```diff
@@ -181,4 +181,11 @@
       items.any? { |item| do_something[0..item] == item }
     RUBY
   end
+
+  it 'does not register an offense when the block is empty' do
+    expect_no_offenses(<<~RUBY)
+      items.any? { |item| }
+      items.any? { }
```

Comment:
Can you either keep only one of the test case or separate the test example (`it`)?

Reply:
Yeah. The second one didn't make any trouble, just making sure